src/sbvr-api/sbvr-utils.ts

```diff
@@ -1270,7 +1270,11 @@ const runODataRequest = (req: Express.Request, vocabulary: string) => {
 				>;
 				// Add/check the relevant permissions
 				try {
-					if (abstractSqlModel.tables[resolveSynonym($request)] == null) {
+					const resolvedResourceName = resolveSynonym($request);
+					if (
+						!resolvedResourceName.includes('#canAccess') &&
```

Comment:
Specifically this should be `endsWith` right? And I assume that would also be more performant